src/main/resources/application.yml

```diff
@@ -20,10 +20,6 @@ launchdarkly:
   offline-mode: false
   env: ${LAUNCH_DARKLY_ENV:default}
 
-role-assignment-service:
-  api:
-    url: ${ROLE_ASSIGNMENT_SERVICE_API_URL:http://localhost:4096}
-
 fees:
```

Comment:
This was removed as this is not configured anywhere. Usually we configure the common service urls on the service side so continuing that pattern.